bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTO.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2016 Markus Rathgeb
```

Comment:
Sure, will do that for all copyright headers of new files of this PR.

FYI (why it has been done without others) : As I don't know how to do it correctly I have read 
https://eclipse.org/legal/copyrightandlicensenotice.php

<pre>If the content is subsequently modified and appended to by other copyright owners, the words "and others" are typically appended.</pre>

I assumed this is done on the first modification.
Will do it on creation from now and change this PR.